OpenEdXMobile/src/sharedTest/java/org/edx/mobile/base/http/HttpBaseTestCase.java

```diff
@@ -70,7 +70,8 @@ private String getBaseMockUrl() {
     }
 
     private JsonElement getMockUrlVersion() {
-        final Config.ApiUrlVersionConfig mockUrlVersionConfig = new Config.ApiUrlVersionConfig("v2");
+        final Config.ApiUrlVersionConfig mockUrlVersionConfig = new Config.ApiUrlVersionConfig(
```

Comment:
Shouldn’t we check for non-version APIs as well?
if not then we can remove the non-version APIs moke response from mock_api_responses.properties.

Reply:
If I understood it correctly, the method call is only for those APIs that get their version code from the config file as the rest of them have their version already present in the endpoint string.